### PR TITLE
Improve Left Panel Style and Fill Button

### PR DESCRIPTION
**Pull request by Alecaddd**
The initial PR for the fill button was great, but it needed some improvements especially after using it and reviewing the code more thoroughly.

## What this PR changes
* Removes unnecessary CSS and complex class inheritance. Margins are all managed in GTK.
* Binds the model to the UI in a bidirectional way (see GIF below)
* Restructure the fills box UI to improve the UX of the entire section
* Temporarily hide the blending modes as those will be complex to implement and are mostly used in edge cases.
* Introduces a first version of the `InputField` partial for future implementation of #144 

fill-panel

### CODE REVIEW

File: src/Partials/InputField.vala [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+*/
+
+public class Akira.Partials.InputField : Gtk.Entry {
+    public int chars { get; construct set; }
+    public bool rtl { get; construct set; }
+    public bool icon_right { get; construct set; }
+    public string unit { get; construct set; }
+    public string icon { get; set; }
+
+    public InputField (string unit, int chars, bool icon_right = false, bool rtl = false) {
```
Comment (bilelmoussaoui) [inline src/Partials/InputField.vala]: It would be better to have an enum for the units.

File: src/Partials/LinkedInput.vala [hunk 2]
```diff
@@ -40,7 +40,8 @@ public class Akira.Partials.LinkedInput : Gtk.Grid {
     */
     private bool manually_edited = true;
     
-    public LinkedInput (string label, string unit = "", bool reversed = false, double default_val = 0, double limit = 0.0) {
+    public LinkedInput (string label, string unit = "", bool reversed = false, 
+        double default_val = 0, double limit = 0.0) {
```
Comment (bilelmoussaoui) [inline src/Partials/LinkedInput.vala]: Indentation here seems not correct

File: src/Partials/InputField.vala (hunk 1 above)
Comment (Alecaddd) [reply src/Partials/InputField.vala]: Good call

**Comment by Alecaddd** [comment]
@bilelmoussaoui updated!

**Comment by Alecaddd** [comment]
@albfan and @bilelmoussaoui, can you guys review and approve this if it looks good?

**Comment by Alecaddd** [comment]
I'm merging this as it drastically changes the UI and UX of the colour model, which we want to use for other sections like the border.

**Pull request merged**